.github/workflows/publish.yml

```
@@ -32,7 +32,7 @@ jobs:
       id: publish
       with:
         repository: grafana/website
-        branch: master
+        branch: v7.5.x
```

```suggestion
        branch: master
```
This is the website branch: https://github.com/grafana/website-sync#how-to-use

---

Oops, updated what branch the action  triggers for